app/views/storage_locations/show.html.erb

```diff
@@ -161,3 +165,11 @@
   <li>Save the file as a csv file.</li>
   <li>DO NOT CHANGE ANY ITEM NAME.</li>
 <% end %>
+
+<script>
+  $(document).ready(() => {
+    $('#version_date_select').change((event) => {
```

Comment:
I often use `data-` attributes to handle JS because I think ids often get used to also handling styling. It feels like the `data-` attributes are more aligned with adding JS functionality.

What do you think? Could we name this to something `data-version-data-select` or something?

Reply:
for part 1 - the reason Stimulus uses `data` attributes is simply that it doesn't know anything about the application, since it's a library. :) So it needs to either use classes, or `data`, and in this case `data` is more appropriate.

for part 2 - it's definitely something that *could* happen. We basically can decide to use ID's for CSS and `data` for JavaScript, or IDs for JavaScript and classes for CSS. It's not a huge difference but I do feel that `data` is best used for actual data and not for querying the DOM.

Reply:
Both points make sense! Thanks for providing that insight.

I think there is no clear standard right now and I don't think its worth holding up this PR for that reason :). Would be interesting to work with you to see if we can bring standards in!